apis/grpcapi/external.go

```diff
@@ -542,6 +543,43 @@ func (s *ExternalServer) DeleteAudience(ctx context.Context, req *protos.DeleteA
 	}, nil
 }
 
+func (s *ExternalServer) GetMetrics(_ *protos.GetMetricsRequest, server protos.External_GetMetricsServer) error {
+	ticker := time.NewTicker(time.Second)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-server.Context().Done():
+			return nil
```

Comment:
Should this not return an error?

Reply:
Nah the context gets cancelled when the client closes the connection, so the method just needs to exit